server.go

```diff
@@ -1,24 +1,28 @@
 package main
 
 import (
+	"fmt"
 	"io"
 	"net"
 	"net/http"
 	"os"
+	"time"
 
+	"github.com/gorilla/context"
+
+	"github.com/getlantern/measured"
+
+	"./filters"
 	"./forward"
 	"./httpconnect"
-	"./profilter"
-	"./tokenfilter"
-	"./uidfilter"
 	"./utils"
 )
 
 type Server struct {
 	connectComponent     *httpconnect.HTTPConnectHandler
-	lanternProComponent  *profilter.LanternProFilter
-	tokenFilterComponent *tokenfilter.TokenFilter
-	uidFilterComponent   *uidfilter.UIDFilter
+	lanternProComponent  *filters.LanternProFilter
+	tokenFilterComponent *filters.TokenFilter
+	uidFilterComponent   *filters.UIDFilter
```

Comment:
This is a personal preference. Less modular at the cost of DRY. I prefer modularity. Repetition comes with the Go language and I'm personally fine with it.
Now we have less repeated code, but less clean and homogeneous interfaces. I also thought of this option, of course. Even all middleware in the same package. But why some in the same package and others in a different package? Just because they are small they should be together? I don't find the reasons why this is better, except for removing some repeated code.